Sources/SwiftInfo/main.swift

```diff
@@ -1,24 +1,48 @@
+import ArgumentParser
 import Foundation
 import SwiftInfoCore
 
 let task = Process()
 
-public struct Main {
-    static func run() {
-        let fileUtils = FileUtils()
-        let toolchainPath = getToolchainPath()
-        log("SwiftInfo 2.3.12")
-        if ProcessInfo.processInfo.arguments.contains("-version") {
-            exit(0)
+struct SwiftInfo: ParsableCommand {
```

Comment:
We also need to call this `Swiftinfo` to prevent the help message from parsing it as `swift-info`